main.py

```diff
@@ -92,8 +92,8 @@ def add_all_arguments(parser):
     # log
     parser.add_argument('--save_k_predictions', type=int, nargs='?', const=100, default=0,
                         help='Save top k predictions on test set. k=%(const)s if not specified. (default: %(default)s)')
-    parser.add_argument('--predict_out_path',
-                        help='Path to the an output file holding top k label results (default: %(default)s)')
+    parser.add_argument('--predict_out_path', default='./predictions.txt',
+                        help='Path to the output file holding label results (default: %(default)s)')
```

Comment:
I think quickstart (linear API) should be updated.. Now we tell users to get decision values and check > 0. This can be simplified

Reply:
this one is resolved?? As you have updated quickstart